api/src/migrations/add-branding-doc.js

```diff
@@ -0,0 +1,60 @@
+const db = require('../db-pouch'),
+  logger = require('../logger'),
+  fs = require('fs'),
+  path = require('path'),
+  BRANDING_ID = 'branding',
+  appTitle = 'Medic Mobile';
+
+const logo = {
+  name: 'logo',
+  file: 'medic-logo-light-full.svg',
+  type: 'image/svg+xml',
+  dir: 'logo'
+};
+const favicon = {
+  name: 'favicon',
+  file: 'favicon.ico',
+  type: 'image/x-icon',
+  dir: 'ico'
+};
+
+const attachDocument = (src, doc, resource) => {
+  return new Promise((resolve, reject) => {
+    fs.readFile(src, (err, data) => {
+      if (err) {
+        return reject(err);
+      }
+      db.medic.putAttachment(doc.id, resource.file, doc.rev, new Buffer(data).toString('base64'), resource.type).then(doc => {
+        resolve(doc);
+      });
+    });
+  });
+};
+
+const createDoc = () => {
+  return db.medic.put({
+    _id: BRANDING_ID,
+    title: appTitle,
+    resources: {
+      [logo.name]: logo.file,
+      [favicon.name]: favicon.file
+    }
+  }).then(doc => {
+    const srcLogo = path.join(__dirname, '..', `resources/${logo.dir}`, logo.file);
+    const srcFav = path.join(__dirname, '..', `resources/${favicon.dir}`, favicon.file);
+    attachDocument(srcLogo, doc, logo).then(doc => {
+      attachDocument(srcFav, doc, favicon);
+    });
```

Comment:
Attachments can be added to the doc before it's put so it all gets saved in one `put`. This will save a bit of time and reduce the number of sequences in the couchdb changes feed so it's good practice. We do this in webapp in the AddAttachment service - just copy that code into API and use that instead of using `putAttachment`.